brewtils/rest/easy_client.py

```diff
@@ -236,6 +236,53 @@ def get_logging_config(self, system_name=None, local=False):
         """
         return self.client.get_logging_config(local=local)
 
+    @wrap_response(
+        parse_method="parse_garden", parse_many=False, default_exc=FetchError
+    )
+    def get_garden(self, garden_name, **kwargs):
+        """Get a System
+
+        Args:
+            garden_name: Name of garden to retrieve
+
+        Returns:
+            The Garden
+
+        """
+        return self.client.get_garden(garden_name, **kwargs)
+
+    @wrap_response(parse_method="parse_garden", parse_many=False, default_exc=SaveError)
+    def create_garden(self, garden):
+        """Create a new Garden
+
+        Args:
+            garden (Garden): The Garden to create
+
+        Returns:
+            Garden: The newly-created Garden
+
+        """
+        return self.client.post_gardens(SchemaParser.serialize_garden(garden))
+
+    def remove_garden(self, garden_name):
```

Comment:
A few issues with this one:

* As written, it will return a `Response` object and not a bool.  bool makes more sense in this case and seems like what you intended, so we should fix that.
* The upfront `get_garden` check doesn't appear to work, as the `get_garden` call throws an exception for gardens that don't exist (which should be fixed).
* The fact that you even had to do the upfront `get_garden` check seems like a red flag.  Presumably, `self.client.delete_garden` would return a 404 response for a garden that doesn't exist.  It turns out that it does not.

I *think* the REST API is easy enough to fix, though I'm sure it's not without risk.  I vote we do it anyway because it'll make these changes work the way they are supposed to.